### PR TITLE
Introducing two new helper functions

### PR DESCRIPTION
**Pull request by webii**
…e path, and second one giving only last element of go_package namespace

### CODE REVIEW

File: encoder.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/golang/protobuf/protoc-gen-go/descriptor"
 	"github.com/golang/protobuf/protoc-gen-go/plugin"
 
-	pgghelpers "github.com/moul/protoc-gen-gotemplate/helpers"
+	pgghelpers "github.com/webii/protoc-gen-gotemplate/helpers"
```
Comment (weitzj) [inline encoder.go]: Please fix imports

File: helpers/helpers.go [hunk 2]
```diff
@@ -120,6 +120,8 @@ var ProtoHelpersFuncMap = template.FuncMap{
 	"fieldMapKeyType":         fieldMapKeyType,
 	"fieldMapValueType":       fieldMapValueType,
 	"replaceDict":             replaceDict,
+	"goPkg":			       goPkg,
```
Comment (weitzj) [inline helpers/helpers.go]: Please reformat code (`gofmt`)

File: main.go [hunk 3]
```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/golang/protobuf/protoc-gen-go/plugin"
 	ggdescriptor "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/descriptor"
 
-	pgghelpers "github.com/moul/protoc-gen-gotemplate/helpers"
+	pgghelpers "github.com/webii/protoc-gen-gotemplate/helpers"
```
Comment (weitzj) [inline main.go]: Please fix imports

**Comment by moul** [comment]
commit adapted in #142

**Pull request closed without merging**